uiowahjmjohnsonselt2024/projectdirectory-selt_2024_team_007
# Final UI changes and fixes

### PR Description: Enhanced Chat and Mapping Features  

#### Changes Made:
1. **Improved Chatting and Image Generation**
   - Text is no longer overlaid on generated images, providing a cleaner and more visually appealing result.  
   ![Screenshot of improved image generation](https://github.com/user-attachments/assets/fc8298b8-6536-492d-9d6a-a2dc5bdd01d9)

2. **Mapping Enhancements**
   - Added dynamic map icon support.
   - Enabled variable map sizes for greater flexibility in gameplay scenarios.  
   ![Screenshot of updated map features](https://github.com/user-attachments/assets/0e2b5113-3855-4753-bdad-b65a79f2dcb1)

#### Testing Steps:
1. Generate a series of images through the chat and confirm that no text overlays appear.
2. Test map functionality:
   - Verify icons are displayed correctly for various map objects.
   - Adjust map sizes in settings and ensure correct scaling.

#### Known Issues:
N/A  

Let me know if further edits or details are needed!
